content/en/docs/collector/build-connector.md

```diff
@@ -0,0 +1,545 @@
+---
+title: Building Connector
+linkTitle: How to Create a Connector # Mandatory, make sure that your short title.
+date: 2023-07-11 # Put the current date, we will keep the date updated until your PR is merged
+author: 'Gordon Hamilton' # If you have only one author, then add the single name on this line in quotes.
+
+draft: true # TODO: remove this line once your post is ready to be published
+# canonical_url: http://somewhere.else/ # TODO: if this blog post has been posted somewhere else already, uncomment & provide the canonical URL here.
+spelling:
+  cSpell:ignore spanmetrics servicegraph exampleconnector struct Errorf
+  cSpell:ignore mapstructure pdata mapstructure pmetric ptrace uber gord
+  cSpell:ignore gomod loggingexporter batchprocessor otlpreceiver
+---
+
+## Connectors in OpenTelemetry
+
+This article is most applicable if you already have an instrumented application
+generating some kind of tracing telemetry data and already have an understanding
+of the [OpenTelemetry Collector](/docs/collector).
+
+## What is a Connector?
+
+Each pipeline in the OpenTelemetry Collector acts on one type of telemetry data.
+There may exist the need to process one form of telemetry data into another one,
+but it is required to route the according data to its proper Collector pipeline.
+A connector acts as the means of sending telemetry data between different
+Collector pipelines by connecting them. A connector acts as an exporter to one
+pipeline and a receiver to another.
+
+## Why use a Connector?
+
+There are processors in OpenTelemetry that convert telemetry data of one type
+into another one. A few examples are the spanmetrics processor, as well as the
+servicegraph processor. The spanmetrics processor generates aggregate requests,
+error and duration metrics from span data. T​he servicegraph processor analyzes
+trace data and generates metrics that describe the relationship between the
+services. Both these processors ingest trace data and convert them to metrics
+data. Since pipelines in the OpenTelemetry Collector are for only one type of
+data, it is necessary to convert the trace data from the processor in the traces
+pipeline and send it to the metrics pipeline. Some processors currently transmit
+data by making use of a work-around that follows a bad practice where a
+processor directly exports data after processing. The connector component solves
+the need for this work-around.
```

Comment:
This paragraph makes the reader assume that those processors still exist (they do but they are depreacted), maybe you can rephrase it in a way that says "Historically processors did this and that, which was a bad practice" etc., such that people recognize that this is not the case anymore